src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -144,10 +145,10 @@ async function logGroupsToArray(logGroups: AsyncIterableIterator<CloudWatchLogs.
     return logGroupsArray
 }
 
-export function createFilterpatternPrompter() {
+export function createFilterpatternPrompter(logGroupName: string) {
     return createInputBox({
-        title: 'Keyword Search',
-        placeholder: 'Enter text here',
+        title: `Search Log Group ${truncate(logGroupName, -50)}`,
```

Comment:
example:

<img width="691" alt="image" src="https://user-images.githubusercontent.com/55561878/181921096-c548f2e0-245a-4c5b-9f05-1d36b850f6b0.png">

Reply:
I like the idea of showing only the end. That makes it much easier to see what log group we are dealing without overcrowding the user.